build.sbt

```diff
@@ -32,7 +32,7 @@ lazy val appDependencies: Seq[ModuleID] = compile ++ test
 
 lazy val compile = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-play-25" % "1.2.0"
+  "uk.gov.hmrc" %% "bootstrap-play-25" % "1.2.0-0-g0000000"
 )
```

Comment:
version `bootstrap-play-25" % "1.2.0-0-g0000000` looks a bit odd

Reply:
I assume Toby is still testing now.
Perhaps he will change it to `1.3.0` after `bootstrap-play-25` is released :)